elyra/api/elyra.yaml

```diff
@@ -273,6 +273,51 @@ paths:
         500:
           description: Unexpected error.
 
+  /elyra/pipeline/schedule:
+    post:
+      tags:
+      - pipeline
+      summary: Execute pipelines as batch jobs
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PipelineResource'
+      responses:
+        200:
+          description: The pipeline processor response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PipelineScheduleResponse'
+
+  /elyra/pipeline/export:
+    post:
+      tags:
+      - pipeline
+      summary: Export pipeline
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PipelineExportBodyPost'
+      responses:
+        200:
+          description: The pipeline export response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    description: Pipeline export status
+                  message:
+                    type: string
+                    description: Pipeline export message
```

Comment:
Not a fan of redundant responses - especially since it's not providing any [contextual information](https://github.com/elyra-ai/elyra/blob/master/elyra/pipeline/handlers.py#L52-L53) and, literally, not used by the frontend.

I'd rather see us return `204`, but that's probably a larger group decision.  If the response could contain some resource or **anything** related to this particular request, then `200` and that information would be fine.

@akchinSTC - what _is_ the result of an export?  How does the user access the exported pipeline?

Looking at the request body, it seems like `export_path` is probably what we want here.  If that doesn't include the filename on the request, the response should append the filename.

If that file _could be_ accessed via the content service, then the **best** response would be a URI relative to `/api/contents/`.  This would mean that `export_path` is a subdirectory under the notebook root_dir.

Are users given some help as to where `export_path` will be located - or do they have to type in the full thing?  If always relative to root_dir, then the contents URI is the answer (IMO).

Reply:
an export in the kfp processor returns the path to the file relative to the file browsers current directory. It gets this information from the pipeline editor widget.
https://github.com/elyra-ai/elyra/blob/6fd7a857ca6bcb2cc7010ba11bae244590885b8c/packages/pipeline-editor/src/PipelineEditorWidget.tsx#L525-L532

Reply:
Excellent.  It looks like the export method already [returns the export_path](https://github.com/elyra-ai/elyra/blob/6fd7a857ca6bcb2cc7010ba11bae244590885b8c/elyra/pipeline/processor_kfp.py#L137), so we should just have the handler add that response value to `/api/contents/` as the response.

Here's an example using a pipeline file...
![image](https://user-images.githubusercontent.com/22599560/83824607-d2788480-a68b-11ea-9989-a9b2bcabb255.png)

Reply:
I conflated things a bit in this discussion.  Its the _Location_ header that should represent the resource URI so I think we should set Location to the appropriate path relative to /api/contents... https://github.com/elyra-ai/elyra/blob/master/elyra/metadata/handlers.py#L68

Then, just have the formal response be the `export_path`.

Reply:
thanks... yes i would agree that is better than status/message response... should i go ahead open a new issue/PR for this and work on it there?